Assets/MixedRealityToolkit.SDK/Profiles/DefaultMixedRealityCameraProfile.asset

```diff
@@ -13,14 +13,7 @@ MonoBehaviour:
   m_Name: DefaultMixedRealityCameraProfile
   m_EditorClassIdentifier: 
   isCustomProfile: 0
-  settingsConfigurations:
```

Comment:
Since we're no longer having this by default, are there any docs that explain how to set this up?

Reply:
there will be. i am writing docs after we branch for stabilization (they do not risk breaking the build) :)

Reply:
Going forward it would be worth including the docs with the code change - per our PR guidelines, we should uphold that in all of our PRs:

https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html#documentation-should-be-added-in-the-same-pull-request-as-a-fix--feature